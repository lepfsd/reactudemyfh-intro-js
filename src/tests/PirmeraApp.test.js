    import React from 'react'
    import PrimeraApp from '../PirmeraApp'
    import { shallow } from 'enzyme'
    import toJson from 'enzyme-to-json';

    describe('prueba en <PirmeraApp />', () => {

        test('debe mostrar el mensaje', () => {

            /*const saludo = 'hola soy goku';
            const { getByText } = render(<PrimeraApp saludo={saludo} />);

            expect(getByText(saludo)).toBeInTheDocument(); */
            const saludo = 'hola soy goku';
            const wrapper = shallow(<PrimeraApp saludo={saludo} />);
            expect(wrapper).toMatchSnapshot()

        })
    })