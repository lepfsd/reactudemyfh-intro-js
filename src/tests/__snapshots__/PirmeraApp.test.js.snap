// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prueba en <PirmeraApp /> debe mostrar el mensaje 1`] = `
<Fragment>
  <h1>
     
    hola soy goku
     
  </h1>
  <p>
    algo mas
  </p>
</Fragment>
`;
